src/containerapp/azext_containerapp/_constants.py

```diff
@@ -115,8 +115,8 @@
     timeout: 1800
 """
 
-ACA_BUILDER_BULLSEYE_IMAGE = "mcr.microsoft.com/oryx/builder:debian-bullseye-20231107.2"
-ACA_BUILDER_BOOKWORM_IMAGE = "mcr.microsoft.com/oryx/builder:debian-bookworm-20231107.2"
+ACA_BUILDER_BULLSEYE_IMAGE = "mcr.microsoft.com/oryx/builder:debian-bullseye-20240124.1"
```

Comment:
We have a separate PR for this (https://github.com/Azure/azure-cli-extensions/pull/7217) but, as it didn't get any reviews yet, I'm OK with having everythign in this one. My only suggestion in that case would be: could you please update HISTORY.rst to also mention the builder update (`* Update the source to cloud builder to 20240124.1`)?